windows_install/ssl/ca/README.md

```diff
@@ -0,0 +1 @@
+Place your `chia_ca.crt` and `chia_ca.key` files in this directory for Windows installation.
```

Comment:
Question, this is the Chia CA which is in the chia-blockchain repo, this isnt the specific certs that need to be generated for the user. In fact they are already in this repo. Couldnt we just use those?

Reply:
You're right. In fact this step was a carry over from a previous attempt to automate the reverse proxy setup. Since that is in a separate file, I've removed unneeded references to these.